test/evergreen.yml

```diff
@@ -358,6 +358,14 @@ functions:
           cd build_posix
           ${test_env_vars|} ${make_command|make} VERBOSE=1 check ${smp_command|} 2>&1
         fi
+  "csuite test":
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger/build_posix/test/csuite"
+      script: |
+       set -o errexit
+       set -o verbose
+       ${test_env_vars|} ./${test_exe} ${test_args|} 2>&1
```

Comment:
A nit for the naming of `test_exe`, which makes me think of a Windows-specific executable? Can it be renamed to e.g. "test_app" or "test_exec" instead?

Reply:
I renamed it to `test_binary` as I saw Evergreen was using the following syntax for binaries `xxx_binary`. See the changes in cb9823ae1b605e68ddb7ad62ac4aa98e71db6c76.